backend/src/main/java/de/aljoschanyang/backend/controllers/QuestionController.java

```diff
@@ -0,0 +1,24 @@
+package de.aljoschanyang.backend.controllers;
+
+import de.aljoschanyang.backend.exceptions.NoQuestionsException;
+import de.aljoschanyang.backend.models.QuestionDTO;
+import de.aljoschanyang.backend.services.QuestionService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/api/questions")
+public class QuestionController {
+    private final QuestionService questionService;
+
+    @GetMapping("/{topicId}")
+    public List<QuestionDTO> getQuestionsByTopicId(@PathVariable String topicId) throws NoQuestionsException {
```

Comment:
You don't need to add the throws clause for unchecked exception
```suggestion
    public List<QuestionDTO> getQuestionsByTopicId(@PathVariable String topicId) {
```

Reply:
I changed it